gobblin-metastore/src/main/java/gobblin/metastore/DatabaseJobHistoryStore.java

```diff
@@ -781,4 +784,8 @@ private String constructTableFilter(Table table) {
 
     return sb.toString();
   }
+  
+  private static Calendar getCalendarUTCInstance() {
+    return Calendar.getInstance(TimeZone.getTimeZone("UTC"));
```

Comment:
I think the UTC Timezone object can be put into a constant but I'm not sure about the Calendar object; I've checked some of the internals of ```PreparedStatement#setTimestamp``` and it seems to me that the Calendar object may be mutated which is up to the given driver implementation. E.g:
```com.mysql.jdbc.TimeUtil#jdbcCompliantZoneShift``` :
> // JDBC spec is not clear whether or not this calendar should be immutable, so let's treat it like it is, for safety

What do you think?